### ISSUE TITLE
ModuleNotFoundError: No module named 'polars.utils'

### ISSUE THREAD

**Issue by heury** [Yvictor/polars_ta_extension#9]
After upgrading to polars 1.0, the following error occurred.
Would you guide me how to fix it?

from polars.utils.udfs import _get_shared_lib_location
ModuleNotFoundError: No module named 'polars.utils'

**Comment by sbushmanov** [comment]
```python
#__init.py__
from pathlib import Path
#lib = _get_shared_lib_location(__file__)
lib = Path(__file__).parent.__str__()
```
But anyway, library update is highly appreciated!

**Comment by heury** [comment]
Thanks for your solution

**Comment by firmai** [comment]
It would be greeat to update to Polars 1 and over, this should not be a solution. It should be reopened.

**Comment by Yvictor** [comment]
fix with version 0.1.3
